src/main/java/interface_adapter/set_difficulty/SetDifficultyController.java

```diff
@@ -0,0 +1,27 @@
+package interface_adapter.set_difficulty;
+
+import use_case.set_difficulty.SetDifficultyInputBoundary;
+import use_case.set_difficulty.SetDifficultyInputData;
+
+/**
+ * Controller for the set difficulty use case.
+ */
+public class SetDifficultyController {
```

Comment:
I'm really not sure about a 'difficulty' use case. **Blackjack as a game has no concept of difficulty at all** because the dealer (AI in our case) has explicit rules for when they hit or hold. If I have time to add other games it will probably be simple games like War or Cheat where again there is no concept of playing 'smart' for an AI as they're just games of pure chance. Might need to talk about some ideas here.

Reply:
Initially, difficulty is a parameter for a computer player. I get that the dealer has to play the smartest way according to some handbook or whatever. But if say, a Blackjack game needs at least 1 dealer and 2 players to play, then we would need one common player (the human player) and one computer player (who plays dumb or smart) besides the dealer (which in this case plays the smart way). But I think Blackjack can be played 1 player vs 1 dealer? There's no need for a dumb player...